assets/css/frontend/components/tabs.css

```
@@ -1,3 +1,32 @@
+body:not(.wp-admin) div.wp-block-group.pmk-tabs-table {
+
+	& .tabs-header {
```

```suggestion
	.tabs-header {
```
Nit: `&` isn't required for descendant selectors (can't see a requirement in style guide but correct me if wrong)

:1234: Applies in multiple locations.

---

@peterwilsoncc It's required as per stylelint `selector-nested-pattern` rule.
By the way, nice to meet you! 😅